### PR TITLE
IDE-2165 new/import liferay workspace wizard now ProjectExplorer to H…

### PR DESCRIPTION
**Pull request by AndyWu2015**
…ierarchical

### CODE REVIEW

**Comment by AndyWu2015** [comment]
I choose not to use system event to call the handler but to just copy the code in handler .
There are issues about staticContext , it is set by eclipse render system and hard to call in our code.

File: gradle/plugins/com.liferay.ide.gradle.core/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -17,7 +17,10 @@ Require-Bundle: org.eclipse.core.runtime,
  com.gradleware.tooling.model;bundle-version="0.5.0",
  com.liferay.ide.project.core,
  com.liferay.blade.api,
- org.eclipse.core.jobs
+ org.eclipse.core.jobs,
+ org.eclipse.core.expressions,
+ org.eclipse.ui.navigator,
```
Comment (gamerson) [inline gradle/plugins/com.liferay.ide.gradle.core/META-INF/MANIFEST.MF]: We can't add ui bundles to a core plugin.

Comment (gamerson) [inline gradle/plugins/com.liferay.ide.gradle.core/META-INF/MANIFEST.MF]: try to call the navigator code from gradle.ui plugin

**Comment by gamerson** [comment]
closing please resend!

**Pull request closed without merging**